lib/puppet_x/puppetlabs/aws.rb

```diff
@@ -24,13 +24,24 @@ def to_s
 
     class Aws < Puppet::Provider
       def self.regions
+        #Remove the new AWS Regions which are disabled by default.
+        disabledRegions = [ "ap-southeast-1", "ap-southeast-2", "ap-northeast-1", "ap-northeast-2", "ap-east1", "ap-south-1"]
```

Comment:
minor thing, if its a variable it should have look like this_case or is it a constant ?

Reply:
@tphoney Thank you for the valuable comment. Updating the file.

Reply:
@tphoney Done.Updated the variable name.